pennylane/ftqc/__init__.py

```diff
@@ -25,10 +25,23 @@
     :toctree: api
 
 """
-from pennylane import ExperimentalWarning
-from .parametric_midmeasure import ParametricMidMeasureMP, diagonalize_mcms
+
+from .parametric_midmeasure import (
+    ParametricMidMeasureMP,
+    XMidMeasureMP,
+    YMidMeasureMP,
+    measure_arbitrary_basis,
+    measure_x,
+    measure_y,
+    diagonalize_mcms,
+)
 
 __all__ = [
     "ParametricMidMeasureMP",
+    "XMidMeasureMP",
+    "YMidMeasureMP",
+    "measure_arbitrary_basis",
+    "measure_x",
+    "measure_y",
```

Comment:
I noticed that you mentioned that for ftqc, x- and y-basis measurements are important. Are z-basis measurements not commonplace? If they are, for readability/consistency purposes, do you see any value in doing something like `qml.ftqc.measure_z = qml.measure`?

Reply:
I had just assumed we'd use `qml.measure`, but I like the symmetry of this, I'll add it